<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QDialog" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ArduinoButtonPad</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/media/icon.ico</normaloff>:/media/icon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Serial communication info</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="txtSerialInfo">
        <property name="text">
         <string>Current serial port:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnScanPorts">
        <property name="text">
         <string>Scan COM ports</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpProfiles">
     <property name="title">
      <string>Profiles</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Current profile:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbCurrentProfile"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btnNewProfile">
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSaveProfile">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRenameProfile">
          <property name="text">
           <string>Rename</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDuplicateProfile">
          <property name="text">
           <string>Duplicate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDeleteProfile">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="grpButtonActions">
        <property name="title">
         <string>Button actions</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,1">
           <item>
            <widget class="QLabel" name="lblButton1">
             <property name="text">
              <string>Button 1:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtAction1">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
           <item>
            <widget class="QLabel" name="lblButton2">
             <property name="text">
              <string>Button 2:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtAction2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
           <item>
            <widget class="QLabel" name="lblButton3">
             <property name="text">
              <string>Button 3:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtAction3">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,1">
           <item>
            <widget class="QLabel" name="lblButton4">
             <property name="text">
              <string>Button 4:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtAction4">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
           <item>
            <widget class="QLabel" name="lblButton5">
             <property name="text">
              <string>Button 5:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtAction5">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkCycleProfiles">
        <property name="text">
         <string>Use button 5 to cycle through profiles</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Other settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="chkShowWinOnStartup">
        <property name="text">
         <string>Show this window on startup</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkAutorunAtStartup">
        <property name="text">
         <string>Autorun at Windows startup</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnQuit">
       <property name="text">
        <string>Quit application</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close dialog</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
